mcstatus/scripts/mcstatus.py

```diff
@@ -70,6 +71,43 @@ def status():
     )
 
 
+@cli.command(short_help="all available server information in json")
+def json():
+    """
+    Prints server status and query in json. Supported by all Minecraft
+    servers that are version 1.7 or higher.
+    """
+    data = {'online': False}
+    # Build data with responses and quit on exception
+    try:
+        ping_res = server.ping()
+        data['online'] = True
+        data['ping'] = ping_res
+        data = {
+            'online': True,
+            'ping': ping_res,
+        }
```

Comment:
I notice that `'online'` and `'ping'` are repeated above. Please remove lines 86-89, since lines 84-85 already do the same.